sa/sa_test.go

```diff
@@ -774,3 +774,52 @@ func TestAddIssuedNames(t *testing.T) {
 		t.Errorf("Wrong args: got\n%#v, expected\n%#v", e.args, expectedArgs)
 	}
 }
+
+func TestDeactivateAuthorization(t *testing.T) {
```

Comment:
Can you add a test for deactivating a pending authz? I think this test only deactivates the finalized authz.